### DISCUSSION TITLE
save records on another server

### DISCUSSION THREAD

**Discussion by izdiadit** [mynaparrot/plugNmeet-server#212] [General]
How can I change the record location to be saved on another server automatically after recording finished

**Comment by jibon57** [comment]
You'll require to mount it in recorder server & change location:

https://github.com/mynaparrot/plugNmeet-server/blob/662165d30c4964a1db37864025fe721c2c61f183/config_sample.yaml#L80


https://github.com/mynaparrot/plugNmeet-recorder/blob/bf4c958d344242fbb6a2c2722e215c28b8b3da0b/config_sample.yaml#L9
